README.md

```diff
@@ -27,7 +27,7 @@ Currently, the minimum supported Rust version is `1.48.0`.
 ## Documentation
 
 - [Examples](https://github.com/gtk-rs/gtk4-rs/tree/master/examples)
-- Book [Stable]/[Development](https://gtk-rs.org/gtk4-rs/book/git/index.html)
+- Book [Stable]/[Development](https://gtk-rs.org/gtk4-rs/git/book/)
```

Comment:
I noticed that there are other broken links (see the per crate readmes)

Reply:
Yes, indeed.

I now fixed the remaining ones